src/Intuition.php

```diff
@@ -1417,10 +1419,10 @@ public function listToText( $l ) {
 	 * Use this when you've changed the cookies and don't want to refresh
 	 * for it to be applied.
 	 *
-	 * @return true
+	 * @return bool Always true.
 	 */
 	public function refreshLang() {
-		$this->initLangSelect();
+		$this->initLangSelect( false );
```

Comment:
I recognise that it was confusing, given it had no default, and wasn't passed. However, just to confirm, this wasn't causing a bug right?

If so, I'd prefer to keep it omitted and instead make null the explicit default.

Reply:
Yes, sure thing. It was giving an error, but fixing it with a default value works just as well.